JapaneseFilter/sections/general_elemhide.txt

```diff
@@ -62,6 +62,8 @@
 ##.ad_textlink_box
 ##.ad_thumbnail_header
 ##.adbox
+! https://github.com/AdguardTeam/AdguardFilters/issues/133617
+##.adBox
```

Comment:
Why do you add this rule as generic?
It should be something like that and you need to add it to the specific.txt.
```
doujinhibiki.net##div[class^="unBox"][class*="ad"]
```

Reply:
> Why do you add this rule as generic? It should be something like that and you need to add it to the specific.txt.
> 
> ```
> doujinhibiki.net##div[class^="unBox"][class*="ad"]
> ```

this rule is just a variation of the `.adbox` rule that is used a lot in Japanese websites. so i think instead of defining a website we should put it in generic

Reply:
> ##.adBox

Better to do it in separate pull request with examples. Btw, this rule does not block leftovers mentioned in the task.

Reply:
I fixed this directly and don't see the need to add this rule to the current site.
@Yuki2718 what do you think about adding the rule above as generic one? Is that really needed?

Reply:
Anyway, we have `##.adBox` in AdGuard Base.

Reply:
As you said `##.adBox` is in optimzed EL so I don't understand how it was an issue - I don't see $generichide even in content_blocker.txt. Could you reproduce?

Reply:
Nope, I can't reproduce it. Then it's ok, thanks for checking this out :)